src/containers/Careers.js

```diff
@@ -98,6 +98,7 @@ class Careers extends Component {
                 ref={`card${index}`}
                 id={`card${card.id}`}
                 key={card.id}
+                color={card.icon_color}
```

Comment:
I think we should try moving this to the `Card` component. Fair enough if it messes up the swing just keep it as is, but it would definitely make more sense in the other file, especially now the styles for the card are split between two files.